plugins/commands/ssh_config/command.rb

```diff
@@ -32,6 +32,11 @@ def execute
           ssh_info = machine.ssh_info
           raise Vagrant::Errors::SSHNotReady if ssh_info.nil?
 
+          if not options[:host].nil? and
+             not options[:host].to_s.eql?(machine.name.to_s)
+            next
+          end
+
```

Comment:
Ah, right, when no `--host` is provided, all should be printed, so only `next`ing when `options[:host]` is not `nil`. Still getting my :coffee: in. :smiley: 

Since code style in general prefers symbols over keywords, perhaps:

```ruby
if options[:host] && options[:host].to_s != machine.name.to_s
  next
end
```

and thanks for the quick followup!